docs/data/material/getting-started/overview/overview.md

```diff
@@ -0,0 +1,26 @@
+# Material UI - Overview
+
+<p class="description">Material UI is library of React UI components that implements Google's Material Design.</p>
+
+## Introduction
+
+Material UI is an open-source React component library that implements Google's [Material Design](https://material.io/).
+
+It includes a comprehensive collection of prebuilt components that are ready for use in production right out of the box.
+
+Material UI is beautiful by design, and features a suite of customization options that make it easy to implement your own custom design system on top of our components.
+
+## Advantages of Material UI
+
+- **Ship faster:** thousands of open-source contributors have poured countless hours into these components. Focus on your core business logic—we've got your UI covered.
+- **Beautiful by default:** we're meticulous about our implementation of Material Design, ensuring that every Material UI component meets the highest standards of form and function.
+- **Cross-team collaboration:** Material UI's intuitive developer experience reduces the barrier to entry for back-end devs and less technical designers, empowering teams to collaborate more effectively.
+- **Reliability:** Material UI is almost as old as React itself—its history stretches back to 2014—and we're in this for the long haul. When you build with Material UI, you can count on our community's support for years to come.
+
+## Material UI vs. MUI Base
+
+Material UI and [MUI Base](/base/getting-started/overview/) feature many of the same UI components, but MUI Base does not come with any default styles or styling solutions.
+
+Material UI is _comprehensive_ in that it comes packaged with default styles, and is optimized to work with [Emotion](https://emotion.sh/docs/introduction) (or [styled-components](https://styled-components.com/)) as a styling engine.
```

Comment:
```suggestion
Material UI is _comprehensive_ in that it comes packaged with default styles, and is optimized to work with [Emotion](https://emotion.sh/docs/introduction) (or [styled-components](https://styled-components.com/)).
```
I think it's enough to be implicit.

Reply:
The official optimization of emotion.

```suggestion
Material UI is _comprehensive_ in that it comes packaged with default styles, and is optimized to work with [emotion](https://emotion.sh/docs/introduction) (or [styled-components](https://styled-components.com/)) as a styling engine.
```

Reply:
> The official optimization of emotion.

Really? They capitalize the name all throughout their docs.